packages/icons-react/package.json

```
@@ -11,6 +11,17 @@
   "main": "./lib/index.js",
   "unpkg": "./dist/index.umd.js",
   "module": "./es/index.js",
+  "exports": {
```

Biome (2.4.13)

[error] The key exports was already declared.

(lint/suspicious/noDuplicateObjectKeys)